tracing-opentelemetry/src/lib.rs

```diff
@@ -9,6 +9,15 @@
 //! [OpenTelemetry]: https://opentelemetry.io
 //! [`tracing`]: https://github.com/tokio-rs/tracing
 //!
+//! ### Special Fields
+//!
+//! These fields have specific meaning for `tracing-opentelemetry` and will be
+//! treated as ordinary fields by other layers:
```

Comment:
Should we maybe also note something about reserving the `otel.` prefix for special fields for OpenTelemetry?

Reply:
Added note about prefix reservation 👍